encoding/ipfix/element.go

```diff
@@ -84,154 +75,154 @@ const (
+		elementSpecifier{80, 0}: fieldDefinition{MacAddress, "destinationMacAddress"},
+		elementSpecifier{81, 0}: fieldDefinition{MacAddress, "postSourceMacAddress"},
 		// many, many more, but let's focus on those actually used by my MikroTik
-		ElementSpecifier{176, 0}: FieldDefinition{Uint8, "icmpTypeIPv4"},
-		ElementSpecifier{177, 0}: FieldDefinition{Uint8, "icmpCodeIPv4"},
-		ElementSpecifier{178, 0}: FieldDefinition{Uint8, "icmpTypeIPv6"},
-		ElementSpecifier{179, 0}: FieldDefinition{Uint8, "icmpCodeIPv6"},
+		elementSpecifier{176, 0}: fieldDefinition{Uint8, "icmpTypeIPv4"},
+		elementSpecifier{177, 0}: fieldDefinition{Uint8, "icmpCodeIPv4"},
+		elementSpecifier{178, 0}: fieldDefinition{Uint8, "icmpTypeIPv6"},
+		elementSpecifier{179, 0}: fieldDefinition{Uint8, "icmpCodeIPv6"},
 		// many, many more, but let's focus on those actually used by my MikroTik
-		ElementSpecifier{184, 0}: FieldDefinition{Uint32, "tcpSequenceNumber"},
-		ElementSpecifier{185, 0}: FieldDefinition{Uint32, "tcpAcknowledgementNumber"},
-		ElementSpecifier{186, 0}: FieldDefinition{Uint16, "tcpWindowSize"},
+		elementSpecifier{184, 0}: fieldDefinition{Uint32, "tcpSequenceNumber"},
+		elementSpecifier{185, 0}: fieldDefinition{Uint32, "tcpAcknowledgementNumber"},
+		elementSpecifier{186, 0}: fieldDefinition{Uint16, "tcpWindowSize"},
 		// many, many more, but let's focus on those actually used by my MikroTik
-		ElementSpecifier{189, 0}: FieldDefinition{Uint8, "ipHeaderLength"},
+		elementSpecifier{189, 0}: fieldDefinition{Uint8, "ipHeaderLength"},
 		// many, many more, but let's focus on those actually used by my MikroTik
-		ElementSpecifier{192, 0}: FieldDefinition{Uint8, "ipTTL"},
+		elementSpecifier{192, 0}: fieldDefinition{Uint8, "ipTTL"},
 		// many, many more, but let's focus on those actually used by my MikroTik
-		ElementSpecifier{205, 0}: FieldDefinition{Uint16, "udpMessageLength"},
-		ElementSpecifier{206, 0}: FieldDefinition{Uint8, "isMulticast"},
+		elementSpecifier{205, 0}: fieldDefinition{Uint16, "udpMessageLength"},
+		elementSpecifier{206, 0}: fieldDefinition{Uint8, "isMulticast"},
 		// many, many more, but let's focus on those actually used by my MikroTik
-		ElementSpecifier{224, 0}: FieldDefinition{Uint64, "ipTotalLength"},
-		ElementSpecifier{225, 0}: FieldDefinition{IPv4Address, "postNATSourceIPv4Address"},
-		ElementSpecifier{226, 0}: FieldDefinition{IPv4Address, "postNATDestinationIPv4Address"},
-		ElementSpecifier{227, 0}: FieldDefinition{Uint16, "postNAPTSourceTransportPort"},
-		ElementSpecifier{228, 0}: FieldDefinition{Uint16, "postNAPTDestinationTransportPort"},
+		elementSpecifier{224, 0}: fieldDefinition{Uint64, "ipTotalLength"},
+		elementSpecifier{225, 0}: fieldDefinition{IPv4Address, "postNATSourceIPv4Address"},
+		elementSpecifier{226, 0}: fieldDefinition{IPv4Address, "postNATDestinationIPv4Address"},
+		elementSpecifier{227, 0}: fieldDefinition{Uint16, "postNAPTSourceTransportPort"},
```

Comment:
Codacy found an issue: [avoid magic numbers like '227', create a named constant for it](https://app.codacy.com/gh/mateumann/clouseau/pullRequest?prid=6383124)